src/linagora.esn.profile/app/show/details/profile-show-details.pug

```diff
@@ -18,7 +18,7 @@ sub-header
               .pmo-pic
                 .p-relative.po-card-image(ng-controller='avatarController')
```

Comment:
Is `avatarController` still useful here ?

It'll be interesting to use the local controller for every interaction in this part (this can be done in another issue/PR)

Reply:
yes , we can do this in separate PR , the goal of this one is to update url of avatar